libjsqrc/ethereumjs/.travis.yml

```diff
@@ -1,13 +1,17 @@
 language: node_js
 node_js:
+  - "0.12"
   - "0.11"
   - "0.10"
 before_script:
   - npm install
   - npm install jshint
+  - export DISPLAY=:99.0 
+  - sh -e /etc/init.d/xvfb start
```

Comment:
this is moved into another repo. But we are using **git subtrees** which is pulling whole subrepo into master repository.